views/RandomBeer.jsx

```diff
@@ -0,0 +1,22 @@
+const React = require('react');
+const Layout = require('./Layout');
+const BeerCard = require('./components/BeerCard');
+function RandomBeer(props) {
+  //   console.log('prop', props);
+  return (
+    <Layout>
+      {
+        <BeerCard
+          name={props.randomBeersFromApi[0].name}
+          image_url={props.randomBeersFromApi[0].image_url}
+          description={props.randomBeersFromApi[0].description}
+          tagline={props.randomBeersFromApi[0].tagline}
+          food_pairing={props.randomBeersFromApi[0].food_pairing}
+          brewers_tips={props.randomBeersFromApi[0].brewers_tips}
```

Comment:
You can make this cleaner by taking the [0] index from here and using it in the app.js route.

Check the comment there. :)